admin/packaging.py

```diff
@@ -1195,7 +1228,12 @@ def main(self, top_level=None, base_path=None):
         """
         to_file(self.sys_module.stderr)
 
-        options = BuildOptions()
+        if top_level is None:
+            top_level = FilePath(__file__).parent().parent()
```

Comment:
Is this only expected for testing? Rather than special-casing the code for tests, change the tests to more closely mimic real usage, even if it means adding `FilePath(__file__).parent().parent().parent()` to the tests.